### PR TITLE
[fr] adj avant ou arrière rule

### PR DESCRIPTION
**Pull request by Rebecca-auque**
accord adj avant et arrière

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -10085,6 +10085,17 @@ though, and has since been slightly modified)-->
...
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="ADJ_AVANT_OU_ARRIERE" name="adj avant ou arrière" default="temp_off">
+      <pattern>
+             <token postag="N.*" postag_regexp="yes"/>
+    <marker>
+        <token regexp="yes">avants|arrières</token>
+    </marker>
+</pattern>
+<message>Une erreur d’orthographe semble être détectée.</message>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:10095]: Could you make the message a little more explanatory? Something like "Quand 'arrière' est utilisé en tant qu'adjectif, il est invariable.".

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -10085,6 +10085,17 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="ADJ_AVANT_OU_ARRIERE" name="adj avant ou arrière" default="temp_off">
+      <pattern>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:10089]: Can you confirm that none of the examples found by the rule editor are false positives?
Especially "lignes arrières" seems to be very popular.
Let's put a `<url>` in there, too. How about https://www.larousse.fr/dictionnaires/francais/arri%c3%a8re/5400 ? It says there "L'adjectif est invariable".

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 1 above)
Comment (Rebecca-auque) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:10095]: That's done.

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 2 above)
Comment (Rebecca-auque) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:10089]: It isn't a FP as it means "les lignes qui sont à l'arrière (du terrain...)". I put another url taking into account both "avant" and "arrière" and explaining the rule.

**Comment by udomai** [comment]
merged here: 9799f2771bde3d9d6990dc0679cc9b82240ea8cf

**Pull request closed without merging**